browser/js/common/directives/navbar/navbar.js

```diff
@@ -13,8 +13,9 @@ app.directive('navbar', function ($rootScope, AuthService, AUTH_EVENTS, $log, $s
             };
 
             scope.logout = function () {
-                AuthService.logout().then(function () {
-                   $state.go('home').reload();
+                AuthService.logout()
+                .then(function () {
+                    $state.go($state.current, {}, {reload: true});
```

Comment:
Have you tested this while looking at a page that requires a user account? That was the issue I was getting around when I changed it from `$state.current` the first time